### PR TITLE
Examples for Source.zipN and Source.zipWithN 

### PR DESCRIPTION
**Pull request by johanandren**
Refs #25468

Also: crosslinking for other zippy operators

### CODE REVIEW

File: akka-docs/src/main/paradox/stream/operators/Source-or-Flow/zipAll.md [hunk 1]
```diff
@@ -16,6 +16,13 @@ Combines all elements from each of multiple sources into @scala[tuples] @java[*P
...
+See also:
+
+ * @ref:[zip](zip.md)
+ * @ref:[zipWith](zipWith.md)
+ * @ref:[zipWith](zipWith.md)  
+ * @ref:[zipWithIndex](zipWithIndex.md)
+
 ## Example
 Scala
 :   @@snip [FlowZipSpec.scala](/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowZipSpec.scala) { #zip }
```
Comment (patriknw) [inline akka-docs/src/main/paradox/stream/operators/Source-or-Flow/zipAll.md]: That is a `zip` example, not `zipAll`?

**Review by patriknw — APPROVED** [review]
LGTM, one question about the zipAll example

File: akka-docs/src/main/paradox/stream/operators/Source-or-Flow/zipAll.md (hunk 1 above)
Comment (johanandren) [reply akka-docs/src/main/paradox/stream/operators/Source-or-Flow/zipAll.md]: Didn't change this but I'll look into it.

Comment (johanandren) [reply akka-docs/src/main/paradox/stream/operators/Source-or-Flow/zipAll.md]: Actually, just noticed after removing that sample, the entire page is not about the `zipAll` operator but copy-pasta about `zip`...

Comment (johanandren) [reply akka-docs/src/main/paradox/stream/operators/Source-or-Flow/zipAll.md]: Added example for that as well now

File: akka-docs/src/main/paradox/stream/operators/Source/zipN.md [hunk 2]
```diff
@@ -1,20 +1,40 @@
...
-@@signature [Source.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala) { #zipN }
-
-@@@
+@apidoc[Source.zipN](Source$) { scala="#zipN[T](sources:scala.collection.immutable.Seq[akka.stream.scaladsl.Source[T,_]]):akka.stream.scaladsl.Source[scala.collection.immutable.Seq[T],akka.NotUsed]" java="#zipN(java.util.List)" } 
 
 ## Description
 
-Combine the elements of multiple streams into a stream of sequences.
+Collects one element for every upstream and when all upstreams has emitted one element all of them are emitted downstream as a collection.
+The element order in the downstream collection will be according to the order their respective sources was listed.
```
Comment (chbatey) [inline akka-docs/src/main/paradox/stream/operators/Source/zipN.md]: Optional: Took me a while to grok this sentence, maybe will be the same order as the sources were listed

**Review by chbatey — APPROVED** [review]

File: akka-docs/src/main/paradox/stream/operators/Source/zipN.md (hunk 2 above)
Comment (johanandren) [reply akka-docs/src/main/paradox/stream/operators/Source/zipN.md]: changed!

**Pull request merged**